cli/azd/.vscode/launch.json

```diff
@@ -32,14 +32,14 @@
             "program": "${workspaceFolder}",
             "console": "integratedTerminal",
             "args": [
-                "auth", "login"
+                "init", "-t", "jongio/agent-openai-python-prompty1"
                 //"pipeline", "config", "--principal-name", "foo", "--provider", "github"
                 //"package", "api", "--debug"
                 //"provision"
                 //"up"
                 //"env", "new"
             ],
-            //"cwd": "~/workspace/cwd/path",
+            "cwd": "/workspaces/agent",
             // "env": {
             //     "INT_TAG_VALUE":"1989"
             // }
```

Comment:
Please revert before merge.

Reply:
FYI, there's a comment at the top of this file that explains how to use `git update-index` to ignore changes to this file.  I use that locally so I can muck around with this as much as I want and not worry about committing the changes (the only thing you have to be aware of is if this file ends up changing upstream you'll have to run `git update-index` again so you can get the changes from upstream).